app/presenters/workflow_presenter.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class WorkflowPresenter
+  def initialize(object:, workflow_name:, xml:)
+    @object = object
+    @workflow_name = workflow_name
+    @xml = xml
+  end
+
+  delegate :pid, to: :object
+  delegate :processes, to: :workflow_document
```

Comment:
It looks like you're both delegating and defining `#processes`. Won't one make the other dead code?

Reply:
Ah, good call. I thought I removed this. I suspect it's just noop, because the other one overwrites this.